templates/keycloak_provision

```diff
@@ -20,7 +20,7 @@ if [ $? -eq 0 ]; then echo "Realm exists"; exit 0; fi \
                                         -s clientId=$keycloakClientId \
                                         -s id=$keycloakClientId \
                                         -s 'webOrigins=["http://$cheHost", "https://$cheHost"]' \
-                                        -s 'redirectUris=["http://$cheHost/*", "https://$cheHost/*"]' \
+                                        -s 'redirectUris=["http://$cheHost/dashboard/", "https://$cheHost/dashboard/", "http://$cheHost/workspace-loader/", "https://$cheHost/workspace-loader/", "http://$cheHost/_app/loader.html", "https://$cheHost/_app/loader.html"]' \
```

Comment:
What about an ability to get token from localhost? I kept in mind running local dashboard/workspace-loader instance without modifying redirects uris manually on each new cluster, but I assume it could also help `chectl server:login` do login through web interface, as https://github.com/int128/kubelogin is able to do.

I assume it should be safe to do since it's allowed on OpenShift.io oauth.

Reply:
I am not sure that it is secure, who can confirm that?

Reply:
@ibuziuk Could you help to discover the answer here?

Reply:
Seems `_app/oauth.html` needs to be added here as well https://github.com/eclipse/che/blob/master/assembly/assembly-root-war/src/main/webapp/_app/oauth.html.

What if we even add `_app/*` since we are sure that `_app` is an application hosted by Che Server and we don't really need to list them one by one. @skabashnyuk WDYT?